source/_integrations/azure_data_explorer.markdown

```diff
@@ -0,0 +1,186 @@
+---
+title: Azure Data Explorer
+description: Setup for Azure Data Explorer integration
+ha_category:
+  - History
+ha_release: 20.3
+ha_iot_class: Cloud Push
+ha_codeowners:
+  - '@kaareseras'
+ha_domain: azure_data_explorer
```

Comment:
Check again if the list is complete because I think you are missing a few options 😉

Reply:
What am I missing here?